src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -37,26 +38,22 @@ public EntityAIGoHome(EntityCitizen citizen)
         this.citizen = citizen;
     }
 
-    @Override
-    public void setMutexBits(final int mutexBitsIn)
-    {
-        super.setMutexBits(1);
-    }
-
     /**
      * Checks if the task should be executed.
      * Only try to go home if he should sleep and he isn't home already.
+     *
      * @return true if should execute.
      */
     @Override
     public boolean shouldExecute()
     {
         return citizen.getDesiredActivity() == EntityCitizen.DesiredActivity.SLEEP
-                && !citizen.isAtHome();
+                 && !citizen.isAtHome();
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)